src/test/java/liuni/HealthCheckTest.java

```diff
@@ -0,0 +1,17 @@
+package liuni;
+
+import com.codahale.metrics.health.HealthCheck;
+import liuni.health.LiUniHealthCheck;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import static org.junit.Assert.assertEquals;
+
+public class HealthCheckTest {
```

Comment:
let's have the class name similar to the health check test but adding "Tests" and move it to the proper package